mount/netbsd-lib.pl

```diff
@@ -231,9 +231,9 @@ sub disk_space
 {
 if (&get_mounted($_[1], "*") < 0) { return (); }
 if ($_[0] eq "proc" || $_[0] eq "swap") { return (); }
-if (&backquote_command("df -k ".quotemeta($_[1]), 1) =~
-    /Mounted on\n\S+\s+(\S+)\s+\S+\s+(\S+)/) {
-	return ($1, $2);
+my $out = &backquote_command("df -k ".quotemeta($_[1]), 1);
```

Comment:
Are you sure this OS has the same `df` output as Linux?

Reply:
> Are you sure this OS has the same df output as Linux?

Yes, this regex also works on `df` output in NetBSD!